contracts/UniversalV3Staker.sol

```diff
@@ -94,16 +107,19 @@ contract UniversalV3Staker is IUniversalV3Staker, Multicall {
     /// @param _nonfungiblePositionManager the NFT position manager contract address
     /// @param _maxIncentiveStartLeadTime the max duration of an incentive in seconds
     /// @param _maxIncentiveDuration the max amount of seconds into the future the incentive startTime can be set
+    /// @param _cfNbits cumulative function tree parameter
     constructor(
         IUniswapV3Factory _factory,
         INonfungiblePositionManager _nonfungiblePositionManager,
         uint256 _maxIncentiveStartLeadTime,
-        uint256 _maxIncentiveDuration
+        uint256 _maxIncentiveDuration,
+        uint256 _cfNbits
```

Comment:
I think cfNbits can be determined by predefined tickspace and TICK_MIN/TICK_MAX.

Reply:
will do in the next PR, using mapping from different pools to bits